ui/ducks/metamask/metamask.js

```diff
@@ -198,6 +210,49 @@ export default function reduceMetamask(state = {}, action) {
   }
 }
 
+const toHexWei = (value, expectHexWei) => {
+  return addHexPrefix(expectHexWei ? value : decGWEIToHexWEI(value));
+};
+
+// Action Creators
+export function updateTransactionGasFees({
```

Comment:
This code is just for confirmations where the transaction already exists in state.